packages/containers-shared/src/build.ts

```diff
@@ -65,3 +69,49 @@ export function dockerBuild(
 		});
 	});
 }
+
+// TODO: this should also pull
+export async function buildAllContainers(
```

Comment:
Add comments on the function in this file :)

Interesting things to comment:
- `verifyLocalDevSupported` why this does (does it needs to be in this function to start with)
- can the fn throw, when
- `dockerPath` -> `dockerBinPath`?